deployments/populate-cred.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
```

Comment:
Can this be moved to Jenkinsfile repo?

Reply:
sure will close this PR. and move it to jenkinsfile

Reply:
removed this file